app/views/apcsp/PageMarketing.vue

```diff
@@ -1291,6 +1293,10 @@ p, .text-p {
     filter: drop-shadow(0px 0px 20px #000000);
   }
 
+  .btn-light-shadow {
```

Comment:
any reason for new shadow? .btn-shadow doesn't work well?

Reply:
I changed it because the shadow's colour on the button is slightly more blue in the figma file.